packages/tasit-action/src/TasitAction.js

```diff
@@ -180,6 +182,19 @@ class TransactionSubscription extends Subscription {
           return;
         }
 
+        setTimeout(() => {
+          const currentTime = Date.now();
+          const timedOut =
+            currentTime - this.#lastConfirmationTime >= this.getEventsTimeout();
```

Comment:
We could consider using moment.js for this, but that's probably overkill

Reply:
It's up to you.

Reply:
Let’s keep it as-is for now